docs/Project.toml

```diff
@@ -1,10 +1,12 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 DocumenterCitations = "daee34ce-89f3-4625-b898-19384cb65244"
+GLMakie = "e9467ef8-e4e7-5192-8a1a-b1aee30e663a"
```

Comment:
Should we remove GLMakie.jl as a dependency for now? I don't think GLMakie.jl will build on Buildkite since docs build on Tartarus where GLMakie.jl doesn't build. And we probably want WGLMakie.jl for docs.